docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -398,13 +398,58 @@ saved object ID of the action.
 
 |==============================================
 
+[discrete]
+[[optional-actions-fields-bulk-update]]
+===== Optional `action` fields
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|frequency |String a|Object containing an action’s frequency:
+
+* `summary` (Boolean, required): Defines whether to send notifications as a summary of alerts or for each generated alert.
+
+* `notifyWhen` (String, required`): Defines how often rules run actions. Valid values are:
+
+** `onActiveAlert`: Actions run when the alert is generated.
+** `onThrottleInterval`: Actions run on the specified throttle interval and summarize new alerts generated during that interval.
+
+* `throttle` (String, optional): Defines how often a rule runs reoccurring actions. This custom action interval must be specified in seconds, minutes, hours, or days. For example, `10m` or `1h`. This property is used only if `notifyWhen` is `onThrottleInterval`.
+
+|alerts_filter |Object a|Object containing an action’s conditional filters:
+
+* `timeframe` (Object, optional): Object containing the time frame for when this action can be run.
+
+** `days` (Array of integers, required): List of days of the week on which this action can be run. Days of the week are expressed as numbers between `1-7`, where `1` is Monday and `7` is Sunday. To select all days of the week, enter an empty array.
+** `hours` (Object, required): The hours of the day during which this action can run. Hours of the day are expressed as two strings in the format `hh:mm` in `24` hour time. A start of `00:00` and an end of `24:00` means the action can run all day.
+*** `start` (String, required)
+*** `end` (String, required)
+
+** `timezone` (String, required): An ISO timezone name, such as `Europe/Madrid` or `America/New_York`. Specific offsets such as UTC or UTC+1 will also work, but lack built-in DST.
+
+* `query` (Object, optional): Object containing a query filter which gets applied to an action and determines whether the action should run. 
+** `kql` (String, required): A KQL string.
+** `filters` (Array of objects, required): A filter object, as defined in the `kbn-es-query` package.
+
+|==============================================
+
 [discrete]
 [[throttle-schema-bulk]]
 ===== `throttle` schema
 
-
 `throttle` defines the maximum interval in which a rule's actions are executed. It accepts the following values:
 
+[NOTE]
+
+=========
+
+The rule level `throttle` field is deprecated in {elastic-sec} 8.8 and will remain active for at least the next 12 months.
+
+In {elastic-sec} 8.8 and later, you can use the (<<optional-actions-fields-bulk-update,`frequency`>>) field to define frequencies for individual actions. Actions without frequencies will acquire a converted version of the rule's `throttle` field. In the response, the converted `throttle` setting appears in the individual actions' `frequency` field.
+
+=========
```

Comment:
@peluja1012 @e40pud the first sentence in this note (line 447) is the deprecation notice. Let me know it's correct or if any other changes are needed. Thank you!